tests/Java.Base-Tests/Java.Base-Tests.targets

```diff
@@ -0,0 +1,49 @@
+<Project>
+
+  <ItemGroup>
+    <_BuildJavaBaseTestsJarInputs Include="$(TargetPath)" />
+    <_BuildJavaBaseTestsJarInputs Include="$(MSBuildThisFileFullPath)" />
+    <_BuildJavaBaseTestsJarInputs Include="$(MSBuildThisFileFullPath)" />
+    <_BuildJavaBaseTestsJarInputs Include="java\**\*.java" />
+  </ItemGroup>
+
+  <Target Name="_CreateJavaCallableWrappers"
+      Condition=" '$(TargetPath)' != '' "
+      AfterTargets="Build"
+      Inputs="@(_BuildJavaBaseTestsJarInputs)"
+      Outputs="$(IntermediateOutputPath)java\.stamp">
+    <RemoveDir Directories="$(IntermediateOutputPath)java" />
+    <MakeDir Directories="$(IntermediateOutputPath)java" />
+    <ItemGroup>
+      <!-- I can't find a good way to trim the trailing `\`, so append with `.` so we can sanely quote for $(_Libpath) -->
+      <_RefAsmDirs Include="@(ReferencePathWithRefAssemblies->'%(RootDir)%(Directory).'->Distinct())" />
+    </ItemGroup>
+    <PropertyGroup>
+      <_JcwGen>$(ToolOutputFullPath)/jcw-gen.dll</_JcwGen>
```

Comment:
Does this need to be quoted? I don't know if `$(ToolOutputFullPath)` could ever have spaces in it.